docs/changelog.md

```diff
@@ -10,14 +10,13 @@ sidebar_label: Changelog
 
 ### Known Bugs
 
- * Loading streams created by older versions cause Memgraph to crash. This is
-   going to be fixed in the next version. For now, the only possible workaround
-   involves **deleting the existing streams**. If the streams cannot be
-   recreated easily, then our advice is not to upgrade to this version. The
-   streams can be deleted by the `DROP STREAM` query in the old versions of
-   Memgraph. After upgrading to this version, the `streams` directory has to be
-   deleted manually from Memgraph's data directory (on Debian-based systems, it
-   is `/var/lib/memgraph` by default).
+* Loading streams created by older versions cause Memgraph to crash. The only
```

Comment:
Shouldn't we also mention something as part of breaking changes? We broke the persisted streams.

It is hard to differentiate the breaking change (we cannot load old streams) and the bug (while memgraph tries to load old streams, it crashes), but I think we should separate them.

Reply:
We could separate them, but we decided not to fix the general problem, only the unfortunate side effect which is crash instead of silent warning and skip. So I would push everything to Breaking changes.